op-program/client/l2/engineapi/l2_engine_api.go

```diff
@@ -350,6 +350,25 @@ func (ea *L2EngineAPI) forkchoiceUpdated(ctx context.Context, state *eth.Forkcho
 	return valid(nil), nil
 }
 
+func toGethWithdrawals(payload *eth.ExecutionPayload) []*types.Withdrawal {
+	if payload.Withdrawals == nil {
+		return nil
+	}
+
+	result := []*types.Withdrawal{}
```

Comment:
```suggestion
	result := make([]*types.Withdrawal, 0, len(payload))
```
leads to more efficient memory alloc

Reply:
👍 Added this in the latest commit.